### PR TITLE
Patterns working [wait - dependent on PL PR]

### PR DESCRIPTION
**Pull request by thisisdano**
This adds a baseline pattern for the breadcrumb nav to the PL, styled in utility classes that could be translated into named classes and stylesheet additions if desired.

screen shot 2016-12-19 at 9 51 35 pm

- - -

screen shot 2016-12-19 at 9 51 14 pm

- - -

screen shot 2016-12-19 at 9 51 52 pm

- - -

screen shot 2016-12-19 at 9 51 44 pm

### CODE REVIEW

**Comment by msecret** [comment]
From a documentation standpoint, theres a lot of repetition between the 3 files. Why do we need 3 separate ones to show the different states?

**Comment by thisisdano** [comment]
Ha, agreed! — The simple answer is cos I don't know how to write it more efficiently yet! I just wanted to make sure everything worked before figuring out how to use data to build variants — which I'm not great at, and it's slow going for me.

I consider this a WIP — anyone should modify it to make it better. 

How would you rebuild it?

**Comment by thisisdano** [comment]
I don't know how to show the different states without variants, tho — whether full html (repetitive) as I've done, or as data-driven, which would take me more time to figure out...

**Comment by msecret** [comment]
Hmm Ok, I'll try and take a look at that when I get the chance.

**Comment by thisisdano** [comment]
Made some changes here, making a new category called "prototypes" and putting the breadcrumb bar in there — this way we'll have more distinction between what we see as a development pattern and what's not yet in development.

File: docs/src/components/component/breadcrumb-nav/bc-app-view/bc-app-view--app-closed.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<nav class="bg-white p4 mn2">
+  <div class="row">
+    <div class="col-fit h8 row-end">
+      <div class="h6 row-center-x">
+        {% render '@icon-square--home-active-large'%}
```
Comment (line47) [inline docs/src/components/component/breadcrumb-nav/bc-app-view/bc-app-view--app-closed.html]: I'd suggest removing the sizing here `large` in favor of letting the block the icon is in define the size. I'd recommend this is other cases in these HTML files.

File: docs/src/components/component/breadcrumb-nav/bc-app-view/bc-app-view--app-closed.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      </div>
+    </div>
+    <div class="col-fit h8 row-end">
+      <div>
+        <div class="h2 sans-s1 fw500 uppercase ls-loose">
+          App
+        </div>
+        <div class="row-center-x">
+          <div class="col-fit mr2 h6 row-center-x">
+            {% render '@icon-square--app-ok-large'%}
```
Comment (line47) [inline docs/src/components/component/breadcrumb-nav/bc-app-view/bc-app-view--app-closed.html]: I'd suggest removing the sizing here `large` in favor of letting the block the icon is in define the size.

File: docs/src/components/component/breadcrumb-nav/bc-app-view/bc-app-view--app-closed.html (hunk 1 above)
Comment (thisisdano) [reply docs/src/components/component/breadcrumb-nav/bc-app-view/bc-app-view--app-closed.html]: I think I just don't know how to to that in the templates — I agree tho, that the icon size could/should be defined in its parent component. This prototype doesn't use subcomponents, tho (outside of the icon component).

Unless I misunderstand the comment.

Comment (thisisdano) [reply docs/src/components/component/breadcrumb-nav/bc-app-view/bc-app-view--app-closed.html]: Additionally — to be very clear about the purpose of my prototypes, it's unlikely that they can or should be a good example of template logic — they're almost exclusively to demonstrate proper display

**Comment by msecret** [comment]
@thisisdano fractal has support for this through it's [statuses](http://fractal.build/guide/core-concepts/statuses) functionality, I'd prefer to use that then add another folder and category.

**Comment by thisisdano** [comment]
Well I'd been using that functionality, but it still didn't seem to be getting the point across. Still, I don't have a strong opinion. I guess I prefer a new folder but who cares really

**Comment by thisisdano** [comment]
I don't want to split hairs in this thing — I just want to get something up and running

**Comment by thisisdano** [comment]
As long as we're merging into `dw-gtd`, I've added requested changes from that PR into this one. Once this branch is merged into `dw-gtd`, those changes should be there.

**Comment by msecret** [comment]
Yea I chnaged this PR to merge into `dw-gtd` to make it easier to understand. The only thing: if there are changes in `dw-gtd` that are related to code changes here then there might be merge conflicts.

**Comment by thisisdano** [comment]
Makes sense — that's why I might propose making proposed changes to `dw-gtd` to this branch instead — then we sidestep merge conflicts and also get the desired changes in the PR.

**Comment by msecret** [comment]
Yea that works

**Comment by adborden** [comment]
ah... we should have held off on merging this, it makes #225 much larger!

**Pull request merged**